stubs/openpyxl/openpyxl/worksheet/cell_range.pyi

```diff
@@ -18,7 +18,7 @@ class CellRange(Serialisable):
     @overload
     def __init__(
         self,
-        range_string: str | None,
+        range_string: str,
```

Comment:
`None` is check explicitly *and* is the default: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.1/openpyxl/worksheet/cell_range.py#L48

```suggestion
        range_string: str | None = None,
```

Reply:
That's what the overload below is for. If `range_string` is `None`, you HAVE TO specify min/max col/row. If it's a string, then the min/max col/row params are unused.

Reply:
For some reason I totally missed the overload ...